Helper/Cart.php

```diff
@@ -2215,7 +2196,7 @@ public function collectDiscounts(
             }
         }
         // TODO: move all third party plugins support into filter
-        return $this->eventsForThirdPartyModules->runFilter("collectDiscounts", [$discounts, $totalAmount, $diff], $quote);
+        return $this->eventsForThirdPartyModules->runFilter("collectDiscounts", [$discounts, $totalAmount, $diff], $quote, $paymentOnly);
```

Comment:
Could you please make sure that method collectDiscounts in class I added before support this new parameter?

Reply:
I already did that when creating PR https://github.com/BoltApp/bolt-magento2/pull/920/files#diff-ae5efcb79da3bea225bbd5436a210c55R40